package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@folio/stripes-connect",
-  "version": "7.1.0",
+  "version": "8.1.0",
```

Comment:
This should have gone to 8.0.0. Technically, we haven't formally published anything -- all we've done is make a CI release -- so we could still go to 8.0.0, but we should think very carefully about what will be the least disruptive move for the rest of the stripes ecosystem.